prisma/schema.prisma

```diff
@@ -0,0 +1,36 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+enum BookStatus {
+  NEW
+  VERIFIED
+  BANNED
+  DISABLED
+  PUBLISHED
+}
+
+model Book {
+  id                       Int        @id @default(autoincrement())
+  status                   BookStatus @default(NEW)
+  createdAt                DateTime   @default(now())
+  updatedAt                DateTime   @updatedAt
+  title                    String     @db.VarChar(255)
+  isAvailableForExchanging Boolean    @default(true)
+  author                   Author?    @relation(fields: [authorId], references: [id], onDelete: SetNull)
+  authorId                 Int?
+}
+
+model Author {
+  id    Int    @id @default(autoincrement())
```

Comment:
About Author, should we add more info to this object? dob, short bio,...
Please, think about the real life when implementing OOP.